frontend/src/concepts/pipelines/content/configurePipelinesServer/ObjectStorageSection.tsx

```diff
@@ -42,120 +32,68 @@ type ObjectStorageSectionProps = {
-                        setConfig({
-                          ...config,
-                          objectStorage: {
-                            useExisting: true,
-                            existingName: option,
-                            existingValue: EMPTY_AWS_SECRET_DATA,
-                          },
-                        });
-                        setExistingDataConnectionOpen(false);
-                      }
-                    }}
-                    selections={config.objectStorage.existingName}
-                    menuAppendTo="parent"
-                  >
-                    {dataConnections.map((connection) => (
-                      <SelectOption
-                        key={connection.data.metadata.name}
-                        value={connection.data.metadata.name}
-                      >
-                        {getDataConnectionDisplayName(connection)}
-                      </SelectOption>
-                    ))}
-                  </Select>
-                </FormGroup>
-              </StackItem>
-              <StackItem>{DISABLED_FOLDER_PATH}</StackItem>
-            </Stack>
-          )
-        }
-      />
-      <Radio
-        className="checkbox-radio-fix-body-width"
-        name="data-connection-radio"
-        id="new-data-connection-radio"
-        label="Create new data connection"
-        isChecked={!config.objectStorage.useExisting}
-        onChange={() =>
-          setConfig({
-            ...config,
-            objectStorage: {
-              newValue: EMPTY_AWS_SECRET_DATA,
-              useExisting: false,
-            },
-          })
-        }
-        body={
-          !config.objectStorage.useExisting && (
-            <Stack hasGutter>
-              <StackItem>
-                <AWSField
-                  values={config.objectStorage.newValue}
-                  onUpdate={(newEnvData) =>
-                    setConfig({
-                      ...config,
-                      objectStorage: {
-                        newValue: newEnvData,
-                        useExisting: false,
-                      },
-                    })
+      {PIPELINE_AWS_FIELDS.map((field) =>
+        field.key === 'AWS_ACCESS_KEY_ID' ? (
+          <FormGroup key={field.key} isRequired={field.isRequired} label={field.label}>
+            <InputGroup>
+              <TextInput
+                aria-label={`Field list ${field.key}`}
+                isRequired={field.isRequired}
+                value={
+                  config.objectStorage.newValue.find((data) => data.key === field.key)?.value || ''
+                }
+                placeholder={field.placeholder}
+                onChange={(_, value) => onChange(field.key, value)}
+              />
+              {loaded && !!dataConnections.length && (
+                <Tooltip
+                  content={
+                    'Populate the form with credentials from your selected data connection</div'
```

Comment:
Got a typo at the end.
```suggestion
                    'Populate the form with credentials from your selected data connection.'
```